recipes/aiida-cusp/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "aiida-cusp" %}
+{% set version = "0.1.0b2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fc62a4474503eb12e296f11986c840e0b29ecf4a5ac17acbddd8efbaee3b3a78
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [win or py < 36]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - aiida-core >=1.3.0,<2.0.0
+    - custodian
+    - pymatgen
+    - ase
+
+test:
+  imports:
+    - aiida_cusp
+    - aiida_cusp.calculators
+    - aiida_cusp.data
+    - aiida_cusp.parsers
+    - aiida_cusp.utils
+  commands:
+    - reentry scan -r aiida
```

Comment:
Can you do this during the build to avoid the need to do it after installation? Or is the result dependent on what other plugins are installed?

Reply:
Hi @chrisburr actually the called top-level ``verdi`` command is part of another package (i.e. ``aiida-core``) and only works if the commands exposed by the ``aiida-cusp`` package are discovered correctly by running the ``reentry scan``command after the plugin has been installed.